components/org.wso2.carbon.identity.recovery/src/main/java/org/wso2/carbon/identity/recovery/util/Utils.java

```diff
@@ -212,46 +215,68 @@ public static String doHash(String value) throws NoSuchAlgorithmException {
     }
 
     /**
-     * Set claim to identity store manager.
-     * @param user
-     * @param claimUri
-     * @param value
-     * @throws IdentityStoreException
-     * @throws UserNotFoundException
+     * Add/Update a claim of a user.
+     *
+     * @param uniqueUserId Unique ID of the user.
+     * @param claimUri Claim URI to be added or updated.
+     * @param value Value of the claim.
+     * @param claimDialect Claim dialect of the claim URI. If dialect is null, root dialect will be used.
+     * @throws IdentityStoreException If error occurs while updating the claim.
+     * @throws UserNotFoundException If the user does not exist.
      */
-    public static void setClaimInIdentityStore(User user, String claimUri, String value)
```

Comment:
This is not the correct place for setClaimInIdentityStore and setClaimsInIdentityStore methods. Shall we move this to identity.mgt ? It should be possible to access these outside recovery components.